Bottom: 5d9f75edb0026a357a65b40f6844dc59a4840761
Top:    cd85ba8e3756a543f6d9e278c9a410d424c826bf
Author: scott snyder <sss@karma>
Date:   2020-04-05 00:07:24 -0400

Refresh of cad-cah-smbc

---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/SMBC.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/SMBC.java
index 59fc456..0e855a7 100644
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/SMBC.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/SMBC.java
@@ -49,22 +49,18 @@ public class SMBC extends IndexedComic {
 	@Override
 	protected int parseForLatestId(BufferedReader reader) throws IOException, ComicLatestException {
           String permalink = getPermalinkFromReader (reader);
-          Log.d("SMBC", "test0 permalink " + permalink);
           int lo = 5000;
           int hi = 5500;
           String lhi = getPermalinkForId(hi);
-          Log.d("SMBC", "test1 lhi " + hi + " " + lhi);
 
           while (!lhi.equals (permalink)) {
             hi = (hi-lo)*2 + lo;
             lhi = getPermalinkForId(hi);
-            Log.d("SMBC", "test1a lhi " + hi + " " + lhi);
           }
 
           while (true) {
             int mid = (lo+hi)/2;
             String lmid = getPermalinkForId(mid);
-            Log.d("SMBC", "test2 " + mid + " " + lmid);
             if (lmid.equals (permalink)) {
               hi = mid;
             }
